my-app/src/app/modules/courses-page/courses-page.component.ts

```diff
@@ -0,0 +1,45 @@
+import {Component, ChangeDetectionStrategy, OnInit} from '@angular/core';
+
+import isEqual from "lodash/isEqual";
+
+import {COURSE_DATA} from "@assets/mocks/course-data.mock";
+
+import {Course} from "./core";
+
+import {FilterPipe} from "./core";
```

Comment:
Weird imports) Duplicated path ./core and newlines

Reply:
Thank you for catching it, fixed)